.github/workflows/build-test-publish-docker.yml

```diff
@@ -0,0 +1,74 @@
+name: build-test-publish-docker
+run-name: "Build, test, publish Docker images (branch ${{ github.ref_name }} on ${{ github.event_name }}/${{ github.event.action }} by ${{ github.actor }})"
+
+# This workflow runs the build, test and publish steps for the
+# Keyman build Docker images.
+# It is triggered by a schedule and repository_dispatch event.
+
+on:
+  schedule:
+    #run every Fri, 3 pm UTC
+    - cron: '0 15 * * 5' #
+
+  repository_dispatch:
+    types: ['build-test-publish-docker:*']
+
+env:
+  # Use GitHub registry, if empty
```

Comment:
I don't understand the comment. We always set `REGISTRY`, so how can it be empty?

Reply:
deleted misleading comment, the meaning of "if empty" was "no regisry specified"